dataland-keycloak/dataland_theme/login/resources/scss/password-strength.scss

```diff
@@ -0,0 +1,145 @@
+.d-progressbar {
+  height: 0.25rem;
+  border-radius: 0.25rem;
+}
+
+$password-strength-color-red: #EE1A1A;
+$password-strength-color-yellow: #FBC02D;
+$password-strength-color-green: #4BB917;
+
+.d-password-strength-empty {
+  width: 0 !important;
```

Comment:
Why all the `!important`s here? Using `!important` (at least frequently) is kind of an antipattern imho.

Reply:
Fixed